templates/meta

```diff
@@ -0,0 +1,17 @@
+{
+    "perl" : "6.c",
+    "name" : "Nondescript",
+    "version" : "0.0.1",
+    "description" : "Nondescript.",
+    "authors" : [
+        "John Doe"
+    ],
+    "license" : "GPL-3.0",
```

Comment:
"license" : "Artistic-2.0" ?

Since that is what the vast majority of Perl 6 modules are.

Reply:
But GPL is freedom, and I want to promote freedom :(